svelte-kit-scss/src/lib/components/IssueSearch/IssueSearchListItem/IssueSearchAssignees.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import type { IssueAssignee } from '$lib/interfaces';
+
+  export let assignees: IssueAssignee[];
+  $: assigneesReversed = assignees.slice().reverse();
+  $: assigneesCount = assignees.length;
+
+  let itemSizePx = 0;
+  const computeMarginRightPx = (size: number, index: number): number => {
+    const powerOf2 = assigneesCount - 1 - index;
+    return size / Math.pow(2, powerOf2) - size;
+  };
+</script>
+
+<div class="issue-search-assignees">
+  <div class="zone" bind:clientHeight={itemSizePx}>
+    {#each assigneesReversed as assignee, index}
+      <div
+        class="assignee"
+        style="margin-right: {computeMarginRightPx(itemSizePx, index)}px;"
+        class:last={index === assigneesCount - 1}
+      >
+        <img class="image" src={assignee.avatarUrl} alt={assignee.login} />
+      </div>
+    {/each}
+  </div>
+</div>
+
+<style lang="scss">
+  $size: 1.25em;
+  .issue-search-assignees {
+    position: relative;
+    height: $size;
+    .zone {
+      position: absolute;
+      top: 0;
+      right: 0;
+      height: 100%;
+      &:hover {
+        .assignee {
+          margin-right: 4px !important;
```

Comment:
This is one of the things I like about keeping UI development in storybook, I was trying to see if this could work without the `!important` but I couldn't find an issue with multiple assignees.
In storybook, with some controls configured, I could just add some assignees and see how it looked.
(not a request for change, just food for thought)